src/components/Testimonial.jsx

```diff
@@ -1,8 +1,8 @@
 import TestimonialQuote from "../assets/TestimonialQuote.png"
 
-const Testimonial = ({Name, Position, Quote, Image, Mirrored = false}) => {
+const Testimonial = ({Name, Position, Quote, Image, Mirrored = false, Color = "#FFFFFF" }) => {
         return(
-            <div className={"bg-white flex mx-72 " + (Mirrored ? "flex-row" : "flex-row-reverse")}>
+            <div className={"flex mx-72 " + (Mirrored ? "flex-row " : "flex-row-reverse ") + "bg-[" + Color + "]"}>
                 <div className="basis-3 bg-[#FFA523]"></div>
                 <div className="basis-1/4">
                     <img className="h-full object-cover" src={Image}/>
```

Comment:
On my end theres some slight incosistencies with the heights of these testimonials, don't be afraid to give fixed values for these since responsive design doesn't generally apply to height too much

Reply:
![image](https://github.com/user-attachments/assets/ee2f4388-40f7-404a-ad1c-f221c45d30b7)